src/main/java/com/iprd/fhir/utils/Utils.java

```diff
@@ -574,4 +574,15 @@ public static boolean noneMatchDates(List<Date> dates, Date currentDate) {
 			.noneMatch(date -> date.equals(currentLocalDate));
 	}
 
+	public static Double anonymizedData(double actualValue, double minPercent, double maxPercent) {
```

Comment:
there are chances of this function throwing arithmetic functions, please handle.

Reply:
Ohh I did not know that. Thanks.

Reply:
remove these unused functions.